src/cc65/symentry.c

```diff
@@ -303,7 +303,7 @@ const char* GetSymTypeName (const Type* T)
     if (Sym == 0) {
         return GetBasicTypeName (T);
     }
-    sprintf (TypeName, "%s %s", GetBasicTypeName (T), Sym->Name ? Sym->Name : "<unknown>");
+    sprintf (TypeName, "%s %s", GetBasicTypeName (T), *Sym->Name != '\0' ? Sym->Name : "<unknown>");
```

Comment:
I think the cc65 style is to use `Sym->Name[0]` instead of `*Sym->Name`.

Reply:
Fixed.

Reply:
No problem.